test/basic-querying.test.js

```diff
@@ -153,6 +153,26 @@ describe('basic-querying', function() {
           done();
         });
     });
+    it('should query by ids and null condition', function(done) {
+      // due to some connectors not being assigning null value by default
+      // the query is filtered by the fields which may consist null value
+      // or not exist at all
+      User.findByIds([
+        createdUsers[0].id,
+        createdUsers[1].id],
+        {fields: {name: true, vip: true}}, function(err, users) {
```

Comment:
i'm afraid i don't understand, the test is named: 'should query by ids and null condition' and we are not querying by null value here. I suspect it should be `vip: null` here?